packages/Badge/index.tsx

```diff
@@ -1,12 +1,62 @@
 import React from 'react'
-import { Tag, TagOptions } from '@welcome-ui/tag'
 import { CreateWuiProps, forwardRef } from '@welcome-ui/system'
 
-export type BadgeOptions = Omit<TagOptions, 'size'>
+import * as S from './styles'
+
+export type BadgeOptions = {
+  disabled?: boolean
+  shape?: 'circle' | 'square'
+  size?: 'sm' | 'md'
+  variant?: 'default' | 'primary'
+  // if a number is higher than 99, we replace this number by 99+
+  withNumberAbbreviation?: boolean
+  children: string | number
```

Comment:
We don't want to allow any type of children?
e.g:

```jsx
<Badge>
  <FormattedMessage defaultMessage="New" id="new" />
</Badge>
```

If so, it should be:
```suggestion
  children: React.node
```

Reply:
yes, you can do with the intl hook instead